src/main/java/org/jenkinsci/plugins/workflow/steps/FlowInterruptedException.java

```diff
@@ -64,6 +73,19 @@
     public FlowInterruptedException(@Nonnull Result result, @Nonnull CauseOfInterruption... causes) {
         this.result = result;
         this.causes = Arrays.asList(causes);
+        this.actualInterruption = false;
```

Comment:
Looking through code on GitHub again, maybe the better approach is to flip this, and then just update things like pipeline-build-step to pass `false`?
```suggestion
        this.actualInterruption = true;
```

Reply:
Done in 5cb7d36.